### DISCUSSION TITLE
How to filter items in nav?

### DISCUSSION THREAD

**Discussion by jannejava** [statamic/cms#5323] [Q&A] [answered]
Hi!

This is my first attempt at making a Statamic site. The site is a multi-site for many organizations with multiple languages, so it's a bit complex. 

Is it possible to filter the nav by a variable on the current page? Like if I am standing on a page that has `myvariable=1` and only want to list other pages that have `myvariable=1`

```
{{ myvariable }}  // Prints out the current page's value correctly
{{ nav include_home="true" }}
    {{ if myvariable == myvariable }} // how to compare current page's variable with the one in the nav?
         <a href="{{ url }}"> {{ title }} {{ myvariable }}</a>  // prints out liked page's value correctly. But how to compare?
   {{ endif }}
{{ /nav }}

```

I really appreciate any help you can provide.

/ Jan

**Comment by jasonvarga** [comment, marked as answer]
You can use `{{ page:nav }}` to refer to the top level one.

In your nav tag you can do `{{ if myvariable == page:myvariable }}`

**Reply by jannejava** [reply]
That's it! Thank you Jason, I appreciate it!
